protovalidate/internal/extra_func.py

```diff
@@ -49,9 +50,14 @@ def _validate_hostname(host):
 
 
 def validate_email(addr):
-    if "<" in addr and ">" in addr:
-        addr = addr.split("<")[1].split(">")[0]
+    addrs = getaddresses([addr])
```

Comment:
would [`parseaddr`](https://docs.python.org/3/library/email.utils.html#email.utils.parseaddr) be more appropriate for a single addr?

Reply:
Actually yeah, it _should_ be safe, due to the equality check. I originally switched to `getaddresses` because `parseaddr` succeeds just fine if you pass a group instead of a single address.

Reply:
Done.